modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-cli_{context}"]
+= Configuring a proxy after installation using the CLI
+
+You can use the {product-title} (ROSA) CLI (`rosa`) to add a cluster-wide proxy configuration to an existing ROSA cluster in a Virtual Private Cloud (VPC).
+
+You can also use `rosa` to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA (`rosa`) and OpenShift (`oc`) CLIs on your installation host.
+* You have a ROSA cluster that is deployed in a VPC.
+
+.Procedure
+
+* Edit the cluster configuration to add or update the cluster-wide proxy details:
++
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
++
+While the `additional-trust-bundle-file`, `http-proxy`, and `https-proxy` arguments are optional, if you set a `additional-trust-bundle-file` without either an `http-proxy` or `https-proxy` argument, then the additional trust bundle will still be used for verifying cluster system egress traffic.
++
+[NOTE]
+====
+You should not attempt to change the proxy or additional trust bundle configuration on the cluster directly. These changes must be applied by using the ROSA CLI (`rosa`) or {cluster-manager-first}. Any changes that are made directly to the cluster will be reverted automatically.
+====
+
+.Verification
+
+. List the status of the machine config pools and verify that they are updated:
```

Comment:
hmm, i see now the dependency on `oc` - nevermind the comment about it being a prerequisite 😊

Reply:
> is there a way to do these verification with the UI (or via `rosa`)?
> 
> and if not, is that planned for the future?

My preference would be for this verification step to be CLI based, because the module relates to using the CLI specifically.

@vkareh and @pvasant hi! Can you please advise if this verification step can be done by using the `rosa` CLI instead of `oc`? Thanks!

Reply:
No, we don't have a way of doing this from our API (or CLIs). You will need `oc` for this.

Reply:
Thank you for confirming that. I will subsequently keep the verification `oc` steps and prerequisite.